imports/blocks/related-content/index.js

```diff
@@ -52,55 +53,31 @@ const withContent = graphql(RELATED_CONTENT_QUERY, {
   }),
 });
 
+type IRelatedContent = {
+  content: Object,
+  title: string,
+};
+
+// $FlowMeteor
 @connect()
 @withContent
```

Comment:
@jbaxleyiii @johnthepink am I supposed to leave the decorators there? or do they need change because of flow?